webpack.config.js

```diff
@@ -0,0 +1,37 @@
+const path = require('path');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+
+module.exports = {
+  mode: 'development',
+  entry: './src/index.js',
```

Comment:
(YGANI): Webpack is configured by default to fetch the index.js file from the src/ folder, so this line is no longer needed, I suggest you remove it.

Reply:
my bad, you're right 👌